dali/base/dadfs.cpp

```diff
@@ -4884,7 +4884,48 @@ protected: friend class CDistributedFilePart;
             calcFileCost(cluster, sizeGB, fileAgeDays, numDiskWrites, numDiskReads, atRestCost, accessCost);
         }
     }
-};
+    // getUnmountedPath - returns false if can't return unmounted path
+    virtual bool getUnmountedPath(unsigned partNum, StringBuffer & path) override
+    {
+        IClusterInfo &cluster = clusters.item(0);
+        const char *planeName = cluster.queryGroupName();
+        if (isEmptyString(planeName))
+            return false;
+        Owned<IStoragePlane> storagePlane = getDataStoragePlane(planeName, false);
+        if (!storagePlane)
+            return false;
+
+        StringBuffer planePrefix(storagePlane->queryPrefix());
+        Owned<IStoragePlaneAlias> alias = storagePlane->getAliasMatch(accessMode);
+        if (alias)
+        {
+            StringBuffer newPlanePrefix(alias->queryPrefix());
+            if (startsWith(path, newPlanePrefix))
+                planePrefix.swapWith(newPlanePrefix);
+        }
+        assertex(startsWith(path, planePrefix));
```

Comment:
question: Is it better to throw an exception or return false?   I suspect this should never happen so an exception is fine.